blocks.js

```diff
@@ -2518,6 +2547,14 @@ BlockMorph.prototype.userMenu = function () {
         'userDestroy'
     );
     menu.addItem(
+      'stringify...',
+        function() {
+            var code = myself.stringify();
+            window.prompt('scratchblocks code for you to copy and paste', code);
```

Comment:
We should make a generic one that has some text about _how_ to copy and paste. C&P is only supported via keyboard shortcuts so it's non-obvious to many. 

Anyway, there's a mostly correct dialog in #1084